app/webpacker/scripts/autocomplete.spec.js

```diff
@@ -34,6 +34,18 @@ describe("Autocomplete", () => {
     });
   });
 
+  describe("getPath", () => {
+    it("should return a path", () => {
+      const path = getPath("/endpoint", "queryString");
+      expect(path).toBe("/endpoint?query=queryString")
+    })
+
+    it("should return an encoded path if unicode is provided", () => {
+      const path = getPath("/endpoint", "Lordswood Girls\u2019 School and Six Form Centr");
+      expect(path).toBe("/endpoint?query=queryString")
```

Comment:
Think the assertion is incorrect here (?)

Reply:
Yes it does, I'm trying to get the js tests running locally atm - I shouldn't really have made the PR yet!